src/main/java/org/infernalstudios/infernalexp/config/CommonConfig.java

```diff
@@ -326,6 +327,15 @@
             .translation(InfernalExpansion.MOD_ID + ".config.tooltip.biomesList")
             .define("biomesList", "");
 
+        builder.push("Dangerous Configs");
+
+        replaceNetherBiomeProvider = builder
+            .comment("Replaces existing worlds' vanilla nether biome provider with ours.\nTHIS CAN BREAK YOUR WORLD, DO NOT TURN ON IF YOU DON'T KNOW WHAT YOU'RE DOING")
+            .translation(InfernalExpansion.MOD_ID + ".config.replaceNetherBiomeProvider")
```

Comment:
Does it need a translation since it's not in the config GUI?

Reply:
it's best for parity with other configs in code, but the lang files omit it because it doesn't show up in the gui